LICENSE.rst

```diff
@@ -2,6 +2,8 @@ The MIT License (MIT)
 
 Copyright (c) 2015 Vanderbilt University
```

Comment:
Is this still the 2015 copyright or should it be updated?

Reply:
I dont think so, i actually think it should be possibly older. I [found a pretty convincing discussion in an issue](https://github.com/google/go-github/issues/823#issuecomment-355795428) that the older year would make more sense. 

I think that would be done in a different PR though. So i will resolve this comment at the moment.